packages/uswds/src/Templates/ButtonTemplates/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { AddButtonProps, IconButtonProps as RjsfIconButtonProps } from "@rjsf/utils";
+import IconButton from "./IconButton"; // Import the generic IconButton
+
+// Define specific button types using the generic IconButton
+const AddButton = (props: AddButtonProps) => (
```

Comment:
We prefer functions to fat-arrows
```suggestion
export function AddButton(props: AddButtonProps) {
```